File: app/src/main/java/com/yashovardhan99/healersdiary/Objects/PatientFeedback.java
package com.yashovardhan99.healersdiary.Objects;

import com.google.firebase.Timestamp;

/**
 * Created by Yashovardhan99 on 28-07-2018 as a part of HealersDiary.
 */
public class PatientFeedback {
    String feedback;
    Timestamp timestamp;
    Boolean verified;
    public PatientFeedback(){
        feedback="";
        timestamp=Timestamp.now();
        verified=false;
    }

    public String getFeedback() {
        return feedback;
    }

    public Boolean getVerified() {
        return verified;
    }

    public Timestamp getTimestamp() {
        return timestamp;
    }
}

File: app/src/main/java/com/yashovardhan99/healersdiary/objects/PatientFeedback.kt
package com.yashovardhan99.healersdiary.objects

import com.google.firebase.Timestamp

import java.text.DateFormat

/**
 * Created by Yashovardhan99 on 28-07-2018 as a part of HealersDiary.
 */
data class PatientFeedback(
        val uid: String,
        val feedback: String,
        val timestamp: Timestamp,
        val verified: Boolean
)
